dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/service/AnonymisingDataService.java

```diff
@@ -0,0 +1,290 @@
+package uk.gov.justice.laa.crime.dces.integration.service;
+
+import lombok.extern.slf4j.Slf4j;
+import net.datafaker.Faker;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+import uk.gov.justice.laa.crime.dces.integration.model.generated.contributions.CONTRIBUTIONS;
+
+import java.nio.ByteBuffer;
+import java.security.SecureRandom;
+import java.util.Optional;
+
+import static uk.gov.justice.laa.crime.dces.integration.utils.DateConvertor.convertToXMLGregorianCalendar;
+
+@Slf4j
+@Service
+public class AnonymisingDataService {
+
+    private static final long DEFAULT_SEED_VALUE = 38;
```

Comment:
With this public information, there's no real gain from using this here.
Could either just remove it, or put it in a secure area like the values files, and reference that here.
Happy with either.